### PR TITLE
Feature/lint dart

### PR DESCRIPTION
**Pull request by tiholic**
Replacement for the mistakenly merged PR #49

### CODE REVIEW

File: analysis_options.yaml [hunk 1]
```diff
@@ -1,2 +1,216 @@
...
 include: package:pedantic/analysis_options.yaml
 
+analyzer:
+
+  strong-mode:
+    implicit-casts: false
+# TODO(zoechi) conflicts with some other rules and is subject to be reworked
+# https://github.com/dart-lang/sdk/issues/33749
+# but in general a useful setting to be enabled eventually
+#    implicit-dynamic: false
```
Comment (QuintinWillison) [inline analysis_options.yaml:14]: Please create an issue for this.

File: analysis_options.yaml [hunk 2]
```diff
@@ -1,2 +1,216 @@
...
+
+#  language:
+# TODO(zoechi) check if this is already available
+# https://github.com/dart-lang/language/blob/master/resources/type-system/strict-raw-types.md
+#    strict-raw-types: true
```
Comment (QuintinWillison) [inline analysis_options.yaml:19]: Please create an issue for this.

**Review by QuintinWillison — APPROVED** [review]
I've only skim read this PR but I love it. ❤️ 
(what's not to love, right? 😁)

Thanks to both @tiholic and @zoechi for your hard work on this one, and my apologies for taking so long to get around to landing it (which I'll do today). 🚀

File: analysis_options.yaml (hunk 1 above)
Comment (tiholic) [reply analysis_options.yaml:14]: #84

File: analysis_options.yaml (hunk 2 above)
Comment (tiholic) [reply analysis_options.yaml:19]: #85

**Pull request merged**